tubemq-manager/src/main/java/org/apache/tubemq/manager/controller/group/GroupController.java

```diff
@@ -0,0 +1,143 @@
+import org.apache.tubemq.manager.controller.topic.request.BatchAddGroupAuthReq;
+import org.apache.tubemq.manager.controller.topic.request.DeleteGroupReq;
+import org.apache.tubemq.manager.controller.topic.request.RebalanceConsumerReq;
+import org.apache.tubemq.manager.controller.topic.request.RebalanceGroupReq;
+import org.apache.tubemq.manager.service.TopicService;
+import org.apache.tubemq.manager.service.MasterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/v1/group")
+@Slf4j
+public class GroupController {
+
+    public Gson gson = new Gson();
+
+    @Autowired
+    private MasterService masterService;
+
+    @Autowired
+    private TopicService topicService;
+
+
+    @PostMapping("")
+    public @ResponseBody TubeMQResult groupMethodProxy(
+        @RequestParam String method, @RequestBody String req) throws Exception {
+        switch (method) {
+            case ADD:
+                return topicService.addConsumer(gson.fromJson(req, BatchAddGroupAuthReq.class));
+            case DELETE:
+                return topicService.deleteConsumer(gson.fromJson(req, DeleteGroupReq.class));
+            case REBALANCE_CONSUMER_GROUP:
+                return topicService.rebalanceGroup(gson.fromJson(req, RebalanceGroupReq.class));
+            case REBALANCE_CONSUMER:
+                return masterService.baseRequestMaster(gson.fromJson(req, RebalanceConsumerReq.class));
+            default:
+                return TubeMQResult.getErrorResult("no such method");
+        }
+    }
+
+    /**
+     * query the consumer group for certain topic
+     * @param req
+     * @return
+     * @throws Exception
+     */
+    @GetMapping("/")
+    public @ResponseBody String queryConsumer(
+        @RequestParam Map<String, String> req) throws Exception {
+        String url = masterService.getQueryUrl(req);
+        return queryMaster(url);
+    }
+
+
+    @PostMapping("/offset")
+    public @ResponseBody TubeMQResult offsetProxy(
+        @RequestParam String method, @RequestBody String req) {
+        switch (method) {
+            case CLONE:
+                return topicService.cloneOffset(gson.fromJson(req, CloneOffsetReq.class));
+            case DELETE:
+                return topicService.deleteOffset(gson.fromJson(req, DeleteOffsetReq.class));
+            default:
+                return TubeMQResult.getErrorResult("no such method");
+        }
+    }
+
+
+
+    @PostMapping("/blackGroup")
```

Comment:
use unified path name, "blackGroup"、"topicConfig" or "black-group"、"topic-config", choose one pattern in all controlles' mapping.